src/command/verify/verify.c

```diff
@@ -586,10 +585,14 @@ verifyUpdateWalFilesMissing(
             ASSERT(strCmp(backup->archiveStart, backup->archiveStop) <= 0);
 
             // We do not process backups from the wrong timeline
-            const String *const anyMissing = missingStart ? missingStart : missingStop;
-            const String *const missingTimeline = strSubN(anyMissing, 0, 8);
-            const String *const backupTimeline = strSubN(backup->archiveStart, 0, 8);
-            bool wrongTimeline = !strEq(missingTimeline, backupTimeline);
+            bool wrongTimeline = false;
+            if (missingStart || missingStop)
```

Comment:
Usually in this file there is an evident comparison to the NULL. Shall we use it instead or the are more examples of short cut?

```suggestion
            if (missingStart != NULL || missingStop != NULL)
```